applications/vessel-history/src/components/Splash.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import Splash from './Splash'
+
+describe('Splash', () => {
```

Comment:
Is this just testing that the component doesn't break? 
Just to understand how it works, if props are need, does it complain?

Reply:
Right, this component does not receive any props. It's just testing that the component rendered matches the snapshot that was generated and saved for it. When it fails we'll have to decide whether we have to update the snapshot becase the change is expected or fix the component to continue rendering what we expected (snapshot)

Reply:
👌